src/ui/invoice/InvoiceShow.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  DateField,
+  ReferenceField,
+  SimpleShowLayout,
+  useRecordContext,
+} from "react-admin";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import Stack from "@mui/material/Stack";
+import { Basket } from "../order";
+
+export const InvoiceShow = () => {
+  const record = useRecordContext();
+  return (
+    <SimpleShowLayout>
+      <Card sx={{ minWidth: 275 }}>
+        <CardContent>
+          <Stack
+            direction="row"
+            spacing={2}
+            justifyContent="space-between"
+            alignItems="center"
+          >
+            <Typography variant="h6">Posters Galore</Typography>
+            <Typography variant="h6">Invoice {record.id}</Typography>
```

Comment:
Why don't you use a `<TextField>` to render te id, instead of relying on `useRecordContext`?

Reply:
Ho yes ! it was from an older commit